pages/design/composite-template-empty.md

```diff
@@ -1,37 +1,40 @@
 ---
+id: 
 name:
 rule_type: composite
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
+accessibility_requirements:
+  wcag20:x.x.x: # (Name of Success Criterion)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
 
-atomic_rules:
+input_rules:
   -
 
 authors:
   -  # (full names as found in `contributors` property in  `package.json` - if not yet listed, please have authors added to the list)
 ---
 
-## Aggregation Definition
-
-### Applicability
+## Applicability
 
 The rule applies to any (??) element ...
```

Comment:
Again: I would suggest "This rule applies to..."

Reply:
I prefer "the rule applies to". Please open an issue if you'd like to discuss this.

Reply:
Also see - https://github.com/act-rules/act-rules.github.io/pull/488#discussion_r284188864